src/components/calcite-tab-title/calcite-tab-title.scss

```diff
@@ -9,29 +9,28 @@
   a {
     outline: none;
     text-decoration: none;
-    color: var(--calcite-tabs-color-active);
```

Comment:
I think these were meant to allow people to theme the tab component. Will wait for @patrickarlt to weigh in.